algorw/planilla.py

```diff
@@ -116,3 +127,38 @@ def _parse_notas(self, rows) -> Dict[str, List[Alumne]]:
                     alulist_by_id.setdefault(grupo, []).append(alu)
 
         return alulist_by_id
+
+    def _parse_repos(self, rows) -> Dict[str, Repo]:
+        """Completa la información sobre repostorios individuales y grupales.
+
+        Este método parsea la hoja Repos y:
+
+          - actualiza todos los campos "repo_indiv" en self._alulist
+          - devuelve un diccionario que mapea legajos y grupos a repositorios.
+        """
+        headers = rows[0]
+        repo_idx = headers.index("Repo")
+        repo2_idx = headers.index("Repo2")
+        grupo_idx = headers.index("Grupo")
+        legajo_idx = headers.index("Legajo")
+        repos_by_id = {}
+
+        for row in islice(rows, 1, None):
+            if legajo := safeidx(row, legajo_idx):
```

Comment:
Si esto da false no debería hacer un continue?

Reply:
Good catch. En realidad el error es que las líneas después del try/except deberían estar indentadas un nivel más. Ahora lo arreglo. 

Lo más correcto hubiera sido un continue con la condición negada (véase, el gran argumento _early-return), pero odio que se necesiten paréntesis en `if not (legajo := safeidx(row, legajo_idx)):`

¡Gracias!